bpfman/src/lib.rs

```diff
@@ -61,11 +61,11 @@ const MAPS_USED_BY_PREFIX: &str = "map_used_by_";
 
 pub(crate) mod directories {
     // When release is made, the dispatcher images should be changed to the
-    // release tag, for example "v0.5.4". After release is made, the tag should
+    // release tag, for example "v0.5.5". After release is made, the tag should
     // be changed back to the "latest" release tag. Note also that these values should
     // be read through the public interfaces provided by config::Config's RegistryConfig
-    pub(crate) const XDP_DISPATCHER_IMAGE: &str = "quay.io/bpfman/xdp-dispatcher:latest";
-    pub(crate) const TC_DISPATCHER_IMAGE: &str = "quay.io/bpfman/tc-dispatcher:latest";
+    pub(crate) const XDP_DISPATCHER_IMAGE: &str = "quay.io/bpfman/xdp-dispatcher:v0.5.5";
```

Comment:
I think we should switch to using a SHA tag on the dispatchers.  If we don't do it in this release, I think we should change it right after the release instead of going back to "latest".  However, if you do it in this release, you won't have to change it after the release.

Reply:
Done